scripts/update.sh

```diff
@@ -27,19 +27,6 @@ if [ $# -eq 1 ]; then
     ROOT_BRANCH="$1"
 fi
 
-# If the repo is not pointing to a branch, the branch name will appear as "HEAD"
-# In that case, we ask the user if we may use the "develop" branch instead
-# If refused, the whole opperation is aborted
-if [ "$ROOT_BRANCH" = "HEAD" ]; then
-    echo "The HEAD is detached from a branch. Should it checkout to develop before proceeding?"
-    select yn in "Yes" "No"; do
-        case $yn in
-            Yes ) ROOT_BRANCH=develop; break;;
-            No ) echo "Aborting..."; exit;;
-        esac
-    done
-fi
-
```

Comment:
Removed because we want to use git TAGs instead branches... Also we don't have developer branch on the repo.